contracts/TablelandTables.sol

```diff
@@ -9,7 +9,9 @@ import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/utils/AddressUpgradeable.sol";
 import "./ITablelandTables.sol";
-import "./ITablelandController.sol";
+import "./Policy.sol";
+import "./ITablelandControllerV1.sol";
+import "./ITablelandControllerV2.sol";
```

Comment:
looking at this now, what about naming the _current_ version `ITablelandController`, ie, no `V2`?